### DISCUSSION TITLE
How to I stream language tweets?

### DISCUSSION THREAD

**Discussion by manoharraon** [tweepy/tweepy#1193] [Questions] [answered]
Hi,

I am trying to stream Portuguese tweets and perform sentiment analysis on those

If I pass in a stream.filter("lang='pt') its not working,

If I use a tweepy.Cursor then I am able to filter tweets based on language but those are not realtime tweets.

I am looking to stream Tweets made in Portuguese and perform sentiment analysis please help

**Comment by Harmon758** [comment, marked as answer]
There's syntactical mistakes with how you're passing the keyword argument, but the name of the kwarg for `Stream.filter` is `languages` anyway:
https://github.com/tweepy/tweepy/blob/cc3c8e7cae73d28b9a75edb7342f89f04da99702/tweepy/streaming.py#L432-L433 It accepts a list of languages, so you would provide it like so: `stream.filter(languages=["pt"])`.
However, note, you'll still need to input either a list of user IDs to follow, using `follow`; a list of keywords to track, using `track`; or a list of locations to track, using `locations`.
See https://developer.twitter.com/en/docs/tweets/filter-realtime/guides/basic-stream-parameters and https://developer.twitter.com/en/docs/tweets/filter-realtime/api-reference/post-statuses-filter.
